psi4/src/read_options.cc

```diff
@@ -1681,6 +1681,8 @@ int read_options(const std::string &name, Options &options, bool suppress_printi
         options.add_int("COSX_RADIAL_POINTS_FINAL", 35);
         /*- Screening criteria for integrals and intermediates in COSX -*/
         options.add_double("COSX_INTS_TOLERANCE", 1.0E-11);
+        /*- Maximum number of SCF iterations to run on on larger (i.e., final) COSX grid -*/
```

Comment:
Maybe be more expansive about the >1, ==1, ==0, <0 values here since this is what gets slurped for the docs.

Reply:
Yeah, that's fair, since COSX_MAXITER_FINAL is a bit more complex than it appears at a first glance. I will do so!

Reply:
Also done! Hopefully, the new description in the code makes sense.